aiagents4pharma/talk2cells/tests/scp_agent/test_scp_agent.py

```diff
@@ -0,0 +1,23 @@
+'''
+Test cases for the search_studies
+'''
+
+# from ..tools.search_studies import search_studies
+from aiagents4pharma.talk2cells.agents.scp_agent import get_app
+from langchain_core.messages import HumanMessage
+
+def test_agent_scp():
```

Comment:
the `traceable` decorators are missing. I assume observability will come in a subsequent PR?